ethernet_init/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+dependencies:
+  idf:
+    version: '>=5.1'
```

Comment:
Is there anything that would prevent this component from working with v5.0 as well?

Reply:
This component should work fine with v5.0. Only I have not yet tested with v5.0 or lower versions.

Reply:
Tested with IDF v5.0